### PR TITLE
Created my studies page

### LINKED ISSUE #44: Implement My Studies page according to design

### PR DESCRIPTION
**Pull request by amberchow8**
-Closes #44

### CODE REVIEW

**Comment by amberchow8** [comment]
Currently the page is displaying all the studies in the database by calling the /studies endpoint which returns all studies, but should we only get the studies from the current user?

File: packages/ui/src/pages/MyStudiesPage/MyStudiesPage.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+</script>
+
+<template>
+  <div class="flex justify-center mt-14 mb-6">
+    <div class="flex flex-row w-3/4">
+      <h1 class="flex w-3/5 text-3xl font-bold">My Studies</h1>
+      <div class="flex w-2/5 justify-end">
+        <RouterLink to="/study">
+          <button
+            class="flex text-sm bg-neutral-300 border border-black rounded-lg py-1 w-30 p-4 h-8 justify-center"
```
Comment (ross3102) [inline packages/ui/src/pages/MyStudiesPage/MyStudiesPage.vue:29]: Wow that's a lot of flex! (And fractional widths) You definitely don't need this much styling, and it has some pretty funky effects when you make the page smaller. Here's a few tips:
* If you have an element with `display: flex` that contains a single child element, you're probably doing something wrong i.e.
```
<div class="flex">
  <div ...>
    ...
  </div>
</div>
```
* See if you can do this layout without any `w-a/b`, you don't need it!
* Think about which elements should be a constant size (`w-n`), and which ones should grow/shrink when the page size changes (`flex-1`), that's what flex is great for!

Comment (ross3102) [reply packages/ui/src/pages/MyStudiesPage/MyStudiesPage.vue:29]: Happy to pair as well if you want

File: packages/ui/src/pages/MyStudiesPage/components/StudyComponent.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+defineProps<{ name: string; description: string }>();
+</script>
+
+<template>
+  <div class="flex flex-row border-b border-neutral-300 w-3/4 h-24">
+    <div class="flex flex-row my-20 justify-left w-4/5 self-center">
```
Comment (ross3102) [inline packages/ui/src/pages/MyStudiesPage/components/StudyComponent.vue:7]: `my-20` that is a lot of margin (80px) lol, is this needed?

Comment (ross3102) [inline packages/ui/src/pages/MyStudiesPage/components/StudyComponent.vue:7]: Same as above, this flex row inside a flex row is a red flag, also `flex-row` is the default behavior for flex anyway so you never need it :)

**Review by ross3102 — CHANGES_REQUESTED** [review]
Left a few comments ranging from minor to major, lmk if you want to pair at some point or give it another shot :)

**Review by ross3102 — APPROVED** [review]

**Pull request merged**